src/h5web/visualizations/line/config.ts

```diff
@@ -22,8 +25,9 @@ export const [useLineConfig] = createPersistableState<LineConfig>(
     setCurveType: (type: CurveType) => set({ curveType: type }),
     showGrid: true,
     toggleGrid: () => set((state) => ({ showGrid: !state.showGrid })),
-    hasYLogScale: false,
-    toggleYLogScale: () =>
-      set((state) => ({ hasYLogScale: !state.hasYLogScale })),
+    scaleType: ScaleType.Linear,
+    setScaleType: (type: ScaleType) => set({ scaleType: type }),
+    isLogDisabled: false,
+    disableLog: (b: boolean) => set({ isLogDisabled: b }),
```

Comment:
As explained in the PR intro, this tells if the log button should be disabled in the `LineToolbar` and is reset at each change of dataset in `LineVis`.